lib/python/gladevcp/drowidget.py

```diff
@@ -41,46 +52,72 @@ class HAL_DRO(Gtk.Label):
         'diameter' : ( GObject.TYPE_BOOLEAN, 'Diameter Adjustment', 'Display Position As Diameter',
                     False, GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT),
         'mm_text_template' : ( GObject.TYPE_STRING, 'Text template for Metric Units',
-                'Text template to display. Python formatting may be used for one variable',
-                "%10.3f", GObject.ParamFlags.READWRITE|GObject.ParamFlags.CONSTRUCT),
+                    'Text template to display. Python formatting may be used for one variable',
+                    "%10.3f", GObject.ParamFlags.READWRITE|GObject.ParamFlags.CONSTRUCT),
         'imperial_text_template' : ( GObject.TYPE_STRING, 'Text template for Imperial Units',
-                'Text template to display. Python formatting may be used for one variable',
-                "%9.4f", GObject.ParamFlags.READWRITE|GObject.ParamFlags.CONSTRUCT),
+                    'Text template to display. Python formatting may be used for one variable',
+                    "%9.4f", GObject.ParamFlags.READWRITE|GObject.ParamFlags.CONSTRUCT),
         'joint_number' : ( GObject.TYPE_INT, 'Joint Number', '0:X  1:Y  2:Z  etc',
                     0, 8, 0, GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT),
         'reference_type' : ( GObject.TYPE_INT, 'Reference Type', '0: Absolute  1:Relative  2:Distance-to-go',
                     0, 2, 0, GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT),
+        'font_family' : ( GObject.TYPE_STRING, 'Font Family', 'The font family of the DRO text: e.g. mono',
+                    "sans", GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT),
+        'font_size' : ( GObject.TYPE_INT, 'Font Size', 'The size of the DRO text from 8 to 96',
+                    8, 96, 40, GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT),
+        'font_weight' : ( GObject.TYPE_STRING, 'Font Weight',
+                    'The size of the DRO text: lighter, normal, bold, or bolder',
+                    "bold", GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT),
+        'unhomed_color' : (Gdk.RGBA.__gtype__, 'Unhomed Color', 'The color of the DRO text when not homed',
+                    GObject.ParamFlags.READWRITE),
+        'homed_color' : (Gdk.RGBA.__gtype__, 'Homed Color', 'The color of the DRO text when homed',
+                    GObject.ParamFlags.READWRITE),
     }
     __gproperties = __gproperties__
 
     def __init__(self, *a, **kw):
+        self.css = Gtk.CssProvider()
+        self.css_text = f"""label {{
+    font-family: sans;
+    font-size: 40px;
```

Comment:
This feels a bit big as default value.

Reply:
The inspector may also be opened at run time so I should mention that as well.

I am not quite sure what you mean with the property names, I named them so they would be consistant with all the other properties. The CSS names are the standard Gtk3 property names.

What font-size do you suggest?